Add a release notes heading for 0.5.0 [ci skip] (#406)
pygae/galgebra: doc/changelog.rst

Changelog
=========

- :release:`0.5.0 <2020.05.27>`

> - :feature:`0` Python 3.8 is now supported and tested.

Reviewer: Also, we drop support for Python 2.7. I don't know if there's a mark for breaking changes.

Reviewer: The PR is https://github.com/pygae/galgebra/pull/232

Author: I think that appears in the [Support] section of the notes already.

Author: Nope, you're right, that's missing.
```suggestion
- :feature:`0` Python 3.8 is now supported and tested. Python 2.7 is no longer supported.
```

Let's just put them in the same bullet point so that users can find it easily.

Reviewer: ```suggestion
- :feature:`232` Python 3.8 is now supported and tested.
- :feature:`212` The support for Python 2.7 is dropped and more features from Python 3.x are used.
```

Reviewer: The above is my story to make dropping 2.7 a feature!

Author: ```suggestion
- :feature:`232` Python 3.8 is now supported and tested.
- :feature:`212` Python 2.7 is no longer supported, allowing more features from Python 3.x to be used.
```

Changelog
=========

> - :release:`0.5.0 <2020.05.27>`

Reviewer: Are we releasing tonight? Or should we make it 05.31 and have some time to wrap it up?

Author: I don't really mind. We can keep updating this date until we actually release.

Changelog
=========

> - :release:`v0.5.0 <2020.05.27>`

Author: ```suggestion
- :release:`0.5.0 <2020.05.27>`
```

>   Can I put text here?

Author: ```suggestion

- :feature:`0` Python 3.8 is now supported and tested
```
Conclusion: No
